pubspec.yaml

```diff
@@ -4,11 +4,11 @@ description: A new flutter project.
 dependencies:
   flutter:
     sdk: flutter
-  image_picker: ^0.0.1
-  google_sign_in: ^0.2.1
-  firebase_auth: ^0.1.0
-  firebase_database: ^0.0.4
-  firebase_storage: ^0.0.1
+  image_picker: 0.4.7
```

Comment:
please include the ^ in these:
^0.4.7 which allows newer versions that 0.4.7 when they come out without updating the yaml file